npm/webpack-preprocessor/package.json

```diff
@@ -59,9 +59,6 @@
     "mocha": "^7.1.0",
     "mockery": "2.1.0",
     "proxyquire": "2.1.3",
```

Comment:
Can we also drop `@babel/plugin-proposal-nullish-coalescing-operator`? It was only used in `use-babel` example AFAIK

Just looking through, maybe out of scope, but looks like these might not be used anymore in this module:
- `@fellow/eslint-plugin-coffee`
- `@typescript-eslint/eslint-plugin`
- `@typescript-eslint/parse`

Might be a few others as well that can now rely on the root level dev deps.

Reply:
I'll take a look later today, I love removing unused deps!

Reply:
me too 😄

Reply:
Cleaned up a few more! Could spend a week+ cleaning up all of the unused deps but I just focused on webpack-preprocessor (and batteries included).

Reply:
There is a bunch of packages to automate removed old deps eg https://www.npmjs.com/package/depcheck. Probably worth doing at some point, should be easy enough to run a few and see what it recommends removing.